### PR TITLE
Load baseUrl from Platform.sh environment variable

### PR DESCRIPTION
**Pull request by GuGuss**

### CODE REVIEW

**Comment by GuGuss** [comment]
#1

**Comment by GuGuss** [comment]
Build.

**Comment by GuGuss** [comment]
Build.

File: angular/app/index.html [hunk 1]
```diff
@@ -71,6 +71,7 @@
     <!-- endbuild -->
 
         <!-- build:js({.tmp,app}) scripts/scripts.js -->
+        <script src="read-write/constants.js"></script>
```
Comment (ralt) [inline angular/app/index.html]: scripts/read-write/constants.js

**Comment by GuGuss** [comment]
Build.

**Pull request closed without merging**